.github/workflows/ci.yaml

```diff
@@ -88,7 +88,9 @@ jobs:
             if [[ ${{ matrix.os }} == "macos-latest" ]]; then
               export PYOPENCL_TEST=port:pthread
             fi
-            python -m pytest --durations=10 --tb=native --junitxml=pytest.xml --doctest-modules -rxsw .
+            python -m pytest --durations=10 --tb=native --junitxml=pytest.xml --doctest-modules -rxsw ./euler
+            python -m pytest --durations=10 --tb=native --junitxml=pytest.xml --doctest-modules -rxsw ./wave
+            python -m pytest --durations=10 --tb=native --junitxml=pytest.xml --doctest-modules -rxsw ./diffusion
```

Comment:
Make this a for loop so as to not repeat the command.

Reply:
Couple of commits for this, but finally i think is ok now.

Reply:
1052c59